Dialogue/Support Partner/sp_010000.csv

```diff
@@ -8,10 +8,10 @@ CHOICE_NOISY#0,"""Dense"""
 CHOICE_SHARP#0,"""Clumsy"""
 CHOICE_BOYISH#0,"""Boyish"""
 CHOICE_NONE#0,"""No change"""
-CHANGE_START#0,"""質問があります。<br>マスターは私の個性を<br>どのように思っているのでしょうか。"""
-CHANGE_PERSONAL#0,"""今の私は、個性に乏しいのでは<br>ないかと思えてきたのです。"""
-CHANGE_ASK#0,"""ゆえにマスター、質問します。<br>私にどのように、なってほしいですか？"""
-CHANGE_OK#0,"""了解しました。<br>マスターの望み通りに……"""
+CHANGE_START#0,"""I have a question. What do you think of my personality, master?"""
+CHANGE_PERSONAL#0,"""I think that I am scarce in personality."""
```

Comment:
"lacking" in personality rather than "scarce", maybe?

Reply:
Sounds like it flows better, personally.